@navikt/ds-css/button.css

```diff
@@ -12,12 +12,12 @@
   border-width: 2px;
 
   /* Padding calculated based on 16px font for 40px tall button */
-  padding: calc(0.5625em - 2px) calc(2em - 2px);
+  padding: calc(0.5625rem - 2px) calc(var(--navds-spacing-8) - 2px);
 }
 
-.navds-button.navds-button--small {
+.navds-button--small {
   /* Padding calculated based on 16px font for 32px tall button */
-  padding: calc(0.3125em - 2px) calc(1.11em - 2px);
+  padding: calc(0.3125rem - 2px) calc(var(--navds-spacing-4) - 2px);
```

Comment:
Jeg endte opp med å endre på noen verdier for å hoverholde systemet.

Reply:
Ser ut som vi misser denne da: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
Minst 44x44px

Reply:
Endret 0.5625rem -> 0.625rem (height: 42px -> height: 44px)
/* Padding calculated based on 18px font for 44px tall button */